api/v1alpha1/workspace_validation.go

```diff
@@ -21,6 +23,9 @@ import (
 const (
 	N_SERIES_PREFIX = "Standard_N"
 	D_SERIES_PREFIX = "Standard_D"
+
+	DefaultLoraConfigMap  = "lora-params"
```

Comment:
DefaultLoraConfigMap->DefaultLoraConfigMap-Template,

"lora-params"-> "lora-params-template"

To make it clear, we need to add keyword template in the names.

Reply:
done